qa/tasks/cephfs/filesystem.py

```diff
@@ -383,9 +387,11 @@ def __init__(self, ctx, fscid=None, create=None):
                 self.name = create
             if not self.legacy_configured():
                 self.create()
-        elif fscid is not None:
-            self.id = fscid
-        self.getinfo(refresh = True)
+                self.getinfo(refresh = True)
+        else:
+            if fscid is not None:
+                self.id = fscid
+            self.name = name
```

Comment:
We don't want a getinfo() here because create is None.

The real problem here is the overloading of the create parameter to also provide the name. I thought adding newfs_raw made more sense than that, but could see it either way.